src/helpers/dataCleaner.tsx

```diff
@@ -0,0 +1,71 @@
+import { getCoordinates, getWeatherData, getAirQualityData} from './apiCalls';
+
+
+export const getAllData = async (query: string) => {
+  const data: any = {};
+
+  // const kelvinToFahrenheit = (kelvin: number) => {
+  //   return (kelvin - 273.15) * 1.8 + 32
+  // }
+
+  // const kelvinToCelcius = (kelvin: number) => {
```

Comment:
These functions are still gonna be used to do the conversions on the display items.